Content.Client/Instruments/MidiParser/MidiInstrument.cs

```diff
@@ -0,0 +1,152 @@
+    REED_ORGAN = 20,
+    ACCORDION = 21,
+    HARMONICA = 22,
+    TANGO_ACCORDION = 23,
+    ACOUSTIC_NYLON_GUITAR = 24,
+    ACOUSTIC_STEEL_GUITAR = 25,
+    ELECTRIC_JAZZ_GUITAR = 26,
+    ELECTRIC_CLEAN_GUITAR = 27,
+    ELECTRIC_MUTED_GUITAR = 28,
+    OVERDRIVEN_GUITAR = 29,
+    DISTORTION_GUITAR = 30,
+    GUITAR_HARMONICS = 31,
+    ACOUSTIC_BASS = 32,
+    FINGERED_ELECTRIC_BASS = 33,
+    PLUCKED_ELECTRIC_BASS = 34,
+    FRETLESS_BASS = 35,
+    SLAP_BASS_1 = 36,
+    SLAP_BASS_2 = 37,
+    SYNTH_BASS_1 = 38,
+    SYNTH_BASS_2 = 39,
+    VIOLIN = 40,
+    VIOLA = 41,
+    CELLO = 42,
+    CONTRABASS = 43,
+    TREMOLO_STRINGS = 44,
+    PIZZICATO_STRINGS = 45,
+    ORCHESTRAL_HARP = 46,
+    TIMPANI = 47,
+    STRING_ENSEMBLE_1 = 48,
+    STRING_ENSEMBLE_2 = 49,
+    SYNTH_STRINGS_1 = 50,
+    SYNTH_STRINGS_2 = 51,
+    CHOIR_AAH = 52,
+    VOICE_OOH = 53,
+    SYNTH_CHOIR = 54,
+    ORCHESTRA_HIT = 55,
+    TRUMPET = 56,
+    TROMBONE = 57,
+    TUBA = 58,
+    MUTED_TRUMPET = 59,
+    FRENCH_HORN = 60,
+    BRASS_SECTION = 61,
+    SYNTH_BRASS_1 = 62,
+    SYNTH_BRASS_2 = 63,
+    SOPRANO_SAX = 64,
+    ALTO_SAX = 65,
+    TENOR_SAX = 66,
+    BARITONE_SAX = 67,
+    OBOE = 68,
+    ENGLISH_HORN = 69,
+    BASSOON = 70,
+    CLARINET = 71,
+    PICCOLO = 72,
+    FLUTE = 73,
+    RECORDER = 74,
+    PAN_FLUTE = 75,
+    BOTTLE_BLOW = 76,
+    SHAKUHACHI = 77,
+    WHISTLE = 78,
+    OCARINA = 79,
+    SQUARE_WAVE_LEAD = 80,
+    SAWTOOTH_WAVE_LEAD = 81,
+    CALLIOPE_LEAD = 82,
+    CHIFF_LEAD = 83,
+    CHARANG_LEAD = 84,
+    VOICE_LEAD = 85,
+    FITHS_LEAD = 86,
+    BASS_LEAD = 87,
+    NEW_AGE_PAD = 88,
+    WARM_PAD = 89,
+    POLYSYNTH_PAD = 90,
+    CHOIR_PAD = 91,
+    BOWED_PAD = 92,
+    METALLIC_PAD = 93,
+    HALO_PAD = 94,
+    SWEEP_PAD = 95,
+    RAIN_EFFECT = 96,
+    SOUNDTRACK_EFFECT = 97,
+    CRYSTAL_EFFECT = 98,
+    ATMOSPHERE_EFFECT = 99,
+    BRIGHTNESS_EFFECT = 100,
+    GOBLINS_EFFECT = 101,
+    ECHOES_EFFECT = 102,
+    SCI_FI_EFFECT = 103,
+    SITAR = 104,
+    BANJO = 105,
+    SHAMISEN = 106,
+    KOTO = 107,
+    KALIMBA = 108,
+    BAGPIPE = 109,
+    FIDDLE = 110,
+    SHANAI = 111,
+    TINKLE_BELL = 112,
+    AGOGO = 113,
+    STEEL_DRUMS = 114,
+    WOODBLOCK = 115,
+    TAIKO_DRUM = 116,
+    MELODIC_TOM = 117,
+    SYNTH_DRUM = 118,
+    REVERSE_CYMBAL = 119,
+    GUITAR_FRET_NOISE = 120,
+    BREATH_NOISE = 121,
+    SEASHORE = 122,
+    BIRD_TWEET = 123,
+    TELEPHONE_RING = 124,
+    HELICOPTER = 125,
+    APPLAUSE = 126,
+    GUNSHOT = 127,
+}
+
+public static class MidiInstrumentExt
+{
+    /// <summary>
+    /// Turns the given enum value into it's string representation to be used in localization.
+    /// </summary>
+    public static string GetStringRep(this MidiInstrument instrument)
+    {
+        return instrument
+            .ToString()
+            .Replace(" ", "_")
+            .Replace("-", "_")
+            .Replace("'", "")
+            .Replace("\"", "")
```

Comment:
I don't understand. None of these enum names contain any of these characters. Why is this needed?